playbook/lib/playbook/kit_base.rb

```diff
@@ -78,8 +80,30 @@ def combined_html_options
       default_html_options.merge(html_options.deep_merge(data_attributes))
     end
 
+    # rubocop:disable Style/OptionalBooleanParameter
+    def pb_content_tag(name, content_or_options_with_block = nil, options = nil, escape = true, &block)
+      options ||= {}
+      combined_options = options.merge(combined_html_options)
+
+      if block_given?
+        content_tag(name, default_options.merge(content_or_options_with_block), combined_options, escape, &block)
+      else
+        content_tag(name, nil, default_options.merge(combined_options), escape)
+      end
+    end
```

Comment:
Options, depending on whether block is given, will naturally differ. https://apidock.com/rails/v7.0.0/ActionView/Helpers/TagHelper/content_tag